src/main/lens-proxy/lens-proxy.injectable.ts

```diff
@@ -5,7 +5,6 @@
 import { getInjectable } from "@ogre-tools/injectable";
```

Comment:
By the end of this PR there are tests that verify this.

Reply:
Yes, but if the unit tests exist when the production code changes for eg. simplicity, it would be **refactoring** instead of **reworking**, making the act more credible. Something to consider in the future. Resolved.